cli/index.js

```diff
@@ -81,4 +431,425 @@ y.command({
     }
 })
 
+// create login command
+y.scriptName("connectme")
+y.command({
+    command: 'login',
+    describe: 'login with an account',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+        password: {
+            describe: 'Password of the account'
+        }
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        login(argv)
+    }
+})
+
+// create guest account command
+y.scriptName("connectme")
+y.command({
+    command: 'guest',
+    describe: 'Creates a guest account ',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+        userID: {
+            describe: 'An user ID',
+            demandOption: true,
+            type: 'string'
+        },
+       picture: {
+           describe: 'A profile picture',
+            demandOption: true,
+            type: 'string'
+        }
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        guest(argv)
+    }
+})
+
+// Logout user
+y.scriptName("connectme")
+y.command({
+    command: 'logout',
+    describe: 'Logout current account ',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+        userID: {
+            describe: 'An user ID',
+            demandOption: true,
+            type: 'string'
+        },
+        
+    },
+    handler(argv) {
+        logOut(argv)
+    }
+})
+
+// recover the account
+y.scriptName("connectme")
+y.command({
+    command: 'recover',
+    describe: 'recover the current account ',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+        userID: {
+            describe: 'An user ID',
+            demandOption: true,
+            type: 'string'
+        },
+        email: {
+            describe: 'An user email',
+            demandOption: true,
+            type: 'string'
+        },

```

Comment:
why are we specifying the email to recover the password? I think it's just the password what's needed here.

Reply:
done